packages/adapter-sqlite/package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "@ai16z/eliza": "workspace:*",
     "@types/better-sqlite3": "7.6.12",
-    "better-sqlite3": "11.6.0",
+    "better-sqlite3": "^11.7.0",
```

Comment:
@stanislawkurzypBD do we need to update it to build our plugin?

Reply:
No we don't, i guess i forgot to rollback this change :face_with_diagonal_mouth: . I'll change it